cmd/notary/keys.go

```diff
@@ -66,8 +66,8 @@ func keysRemove(cmd *cobra.Command, args []string) {
 	//TODO (diogo): Validate Global Unique Name. We probably want to reject 1 char GUNs.
 	gunOrID := args[0]
 
-	// Try to retreive the ID from the CA store.
-	cert, err := caStore.GetCertificateBykID(gunOrID)
+	// Try to retrieve the ID from the CA store.
+	cert, err := caStore.GetCertificateByFingerprint(gunOrID)
```

Comment:
Wish we had a way to optimistically parse and figure it out earlier in the method.